library/Fission/User/Creator/Class.hs

```diff
@@ -65,8 +69,18 @@ instance MonadIO m => Creator (Transaction m) where
 
           Nothing ->
             return <| Left <| openUnionLift User.AlreadyExists
+    where
+      -- parsePW :: MonadIO m => Either DID Text -> m (Either Password.FailedDigest (Maybe Text))
```

Comment:
was getting a horrible ridiculous no good compiler where when I had the type declaration in
```
    • Occurs check: cannot construct the infinite type:
        m ~ ReaderT SqlBackend m
      Expected type: Transaction m (Either Errors UserId)
        Actual type: m (Either
                          (OpenUnion
                             '[User.AlreadyExists, Heroku.AddOn.Error, Password.FailedDigest])
                          (Key User))
........
 ```
which I wasn't able to figure out :confused:  code seems to work fine so I have no clue what's causing it